File: TheaterSuggestions/Java/ScreeningSeatingArrangementSuggestions/src/main/java/org/weaveit/seatingplacesuggestions/SeatingOptionIsNotAvailable.java
package org.weaveit.seatingplacesuggestions;

import java.util.List;

public record SeatingOptionIsNotAvailable(
        int partyRequested,
        PricingCategory pricingCategory
) implements SeatingOption {

    @Override
    public boolean matchExpectation() {
        return false;
    }

    @Override
    public List<SeatingPlace> seats() {
        return List.of();
    }
}

File: TheaterSuggestions/CSharp/ScreeningSeatingArrangementSuggestions/SeatingOptionIsNotAvailable.cs
namespace SeatsSuggestions;

public class SeatingOptionIsNotAvailable : SeatingOptionIsSuggested
{
    public SeatingOptionIsNotAvailable(int partyRequested, PricingCategory pricingCategory)
        : base(partyRequested, pricingCategory)
    {
    }
}
